web/app/js/components/TopModule.jsx

```diff
@@ -202,37 +202,38 @@ class TopModule extends React.Component {
   }
 
   indexTopResult = (d, topResults) => {
+    const topResults_ = topResults;
```

Comment:
This change is no longer required either

Reply:
Yes. You are right. :) Just made the changes.